### ISSUE TITLE
Multiple admins per post

### ISSUE THREAD

**Issue by stanriders** [stanriders/circleforms#171]
There should be possible to assign additional authors to each post

**Comment by B0und** [comment]
Can we access user`s friend list for this? It would be cool to be able to select people from a dropdown or something

**Comment by stanriders** [comment]
Yea, we can
